libs/blocks/figure/figure.css

```diff
@@ -1,5 +1,6 @@
 .figure {
   margin: 40px 0;
+  padding: 20px;
```

Comment:
Adding `20px` padding to `.figure` will create additional space to the container element, as well as all child figures. Given the following DOM structure
```html
<div class="figure figure-list figure-list-2">
  <figure class="figure">
    <picture>
      <source type="image/webp" srcset="./media_1d5a9812d80dc3b39fb0a21e9eac586d6978ee07c.png?width=2000&amp;format=webply&amp;optimize=medium" media="(min-width: 600px)">
      <source type="image/webp" srcset="./media_1d5a9812d80dc3b39fb0a21e9eac586d6978ee07c.png?width=750&amp;format=webply&amp;optimize=medium">
      <source type="image/png" srcset="./media_1d5a9812d80dc3b39fb0a21e9eac586d6978ee07c.png?width=2000&amp;format=png&amp;optimize=medium" media="(min-width: 600px)">
      <img loading="lazy" alt="" src="./media_1d5a9812d80dc3b39fb0a21e9eac586d6978ee07c.png?width=750&amp;format=png&amp;optimize=medium" width="600" height="400">
    </picture>
    <figcaption>
      <em class="caption">In search of</em>
    </figcaption>
  </figure>
  <figure class="figure">
    <picture>
      <source type="image/webp" srcset="./media_195c63834ce5b481e3aefe19ad713c0b1a79ed1d9.png?width=2000&amp;format=webply&amp;optimize=medium" media="(min-width: 600px)">
      <source type="image/webp" srcset="./media_195c63834ce5b481e3aefe19ad713c0b1a79ed1d9.png?width=750&amp;format=webply&amp;optimize=medium">
      <source type="image/png" srcset="./media_195c63834ce5b481e3aefe19ad713c0b1a79ed1d9.png?width=2000&amp;format=png&amp;optimize=medium" media="(min-width: 600px)">
      <img loading="lazy" alt="" src="./media_195c63834ce5b481e3aefe19ad713c0b1a79ed1d9.png?width=750&amp;format=png&amp;optimize=medium" width="600" height="750">
    </picture>
    <figcaption>
      <em class="caption">peace</em>
    </figcaption>
  </figure>
</div>
```
the padding would be applied at the `div.figure` level and then for every `figure.figure` element, introducing more space than needed.
<img width="870" alt="Screenshot 2023-07-17 at 15 23 02" src="https://github.com/adobecom/milo/assets/408175/4beae60a-d9b3-4d8b-8401-555b94c396b8">